mirrord/agent/src/cli.rs

```diff
@@ -36,6 +37,9 @@ pub struct Args {
     #[arg(short = 'p', long, default_value_t = false)]
     pub pause: bool,
 
+    #[arg(long, default_value_t = Features::all())]
+    pub protocol_features: Features,
```

Comment:
Why we would want users to specify that?
Also, why wouldn't it just be v1/v2?

Reply:
It's optional to clamp the agent to a certain version so we can clamp it from the operator if needed

Reply:
but why is it features and not just a version?